helm-chart/kuberay-apiserver/Chart.yaml

```diff
@@ -15,4 +15,4 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.1.0
```

Comment:
I found it odd that this version in master is stale, in a follow-up we should consider whether this should be `nightly` or something next